### PR TITLE
update find_package for console bridge to reflect 3rdparty status

### PR DESCRIPTION
**Pull request by tfoote**
@dirk-thomas @wjwwood please review for emergency release

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,21 +1,22 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(rosconsole_bridge)
 
-find_package(catkin REQUIRED COMPONENTS console_bridge rosconsole)
+find_package(console_bridge REQUIRED)
+find_package(catkin REQUIRED COMPONENTS rosconsole)
 
 include_directories(include ${catkin_INCLUDE_DIRS})
```
Comment (wjwwood) [inline CMakeLists.txt]: Do you need `${console_bridge_INCLUDE_DIRS}` here?

**Comment by wjwwood** [comment]
Otherwise +1

File: CMakeLists.txt [hunk 2]
```diff
@@ -1,21 +1,22 @@
...
-link_directories(${catkin_LIBRARY_DIRS})
+link_directories(${catkin_LIBRARY_DIRS} ${console_bridge_LIBRARY_DIRS})
 
 catkin_package(
-  INCLUDE_DIRS include
-  LIBRARIES ${PROJECT_NAME}
+  INCLUDE_DIRS include ${console_bridge_INCLUDE_DIRS}
+  LIBRARIES ${PROJECT_NAME} ${console_bridge_LIBRARIES}
```
Comment (dirk-thomas) [inline CMakeLists.txt]: This looks unnecessary since `console_bridge` is already passed as a `DEPENDS`?

File: CMakeLists.txt (hunk 1 above)
Comment (dirk-thomas) [inline CMakeLists.txt]: Definitely yes.

**Comment by tfoote** [comment]
Ok removed redundant INCLUDE_DIRS and LIBRARIES in DEPEND

**Pull request merged**